stdstm32-spi-template.h

```diff
@@ -513,9 +513,14 @@ LL_SPI_InitTypeDef SPI_InitStruct = {};
   subghz_init();
 #endif
 
-#if defined SPI$_USE_SPI3 && defined STM32F1
+#if defined STM32F1
+#if defined SPI$_USE_SPI1_PB3PB4PB5
+  LL_GPIO_AF_EnableRemap_SPI1();
+#endif
+#if defined SPI$_USE_SPI3
```

Comment:
I know I'm very nitpicky here, could pl change this one to #ifdef

(just these two, not the if for the stm32)